jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -193,4 +191,23 @@ static FileCollection getInputFiles(File extraDirectory, Project project) {
       return project.files(dependencyFileCollections);
     }
   }
+
+  /**
+   * Validates and converts a {@code String->String} file-path-to-file-permissions map to an
+   * equivalent {@code AbsoluteUnixPath->FilePermission} map.
+   *
+   * @param inputMap the map to convert
```

Comment:
nit: `inputMap` should probably be more descriptive - might be good to even include an example of an entry in this map.